### DISCUSSION TITLE
Monitoring section

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#763] [Ideas]
Hi rejetto,

I wonder if you would consider to create a toggle button to be able to show some kind of history of downloads in the monitoring section of the main menu. If something now is downloaded you can only see that when the download is in progress. I would like that if the download is done that the row with information about will scroll down in some list instead of vanishing. 

My experience is that in the logs most downloads are not registered as downloads. Only some of them will show up there. On the other hand I just saw som download activity (succesfull) in the monitoring section, but again nothing to be found about that in the Logs on the screen. In fact I see hardly any download entries there. Mostly only the images they have to download if they access my server.

Kind regards,
Henk - https://fileserver.ph5hp.nl - https://ph5hp.nl

**Comment by rejetto** [comment]
i see the sense of the log in the monitoring, and i will consider it.

you think the log is not registering things, but that's not unlikely, because there are some options for that.
be sure you check those options.

**Reply by PH5HP** [reply]
Hi rejetto,

Thank you for considering that. Could you tell me what I have to do to get all downloads registered by the logs? In the beginning I saw a lot of downloads there, but for several months now I hardly see any despite of seeing files download in the monitoring section. Did you change something back then?

Kind rergards,
Henk - https://fileserver.ph5hp.nl - https://ph5hp.nl

**Comment by rejetto** [comment]
options that may skip downloads are "don't log address", "log interface loading", "log spam requests".
Surely I changed what's related to this options, within last year.

**Reply by PH5HP** [reply]
Hi,

I discovered that almost no download can be find in the log. So you think it must be an option what is preventing downloads te be logged? 

Below my LOG options as it is at the moment.

![image](https://github.com/user-attachments/assets/a9e5caaa-1c83-4544-8b77-47e1c95428e9)

The 'don't log address' I can not find.

Kind rergards,
Henk - https://fileserver.ph5hp.nl/ - https://ph5hp.nl/

**Comment by rejetto** [comment]
<img width="144" alt="image" src="https://github.com/user-attachments/assets/aeb325a4-cbb0-4b8f-a602-346e4579123c">

as you see, it's by default ignoring same-pc traffic

**Reply by PH5HP** [reply]
Hi,

I do not understand. Does this mean downloads from outside ip's will also not be logged (and seen in the log section)?

Kind regards,
Henk

**Comment by rejetto** [comment]
surely not, i said "ignoring same-pc" but of course it considers other computers.
your settings in the screenshot should log it all.
i made a quick test with your same settings, and i'm getting downloads logged without a problem.
are you totally sure you are missing downloads in the log? make a test making a download yourself.
if you are on the same pc of hfs, to skip the default "don't log address" filter on localhost, just access your server using your LAN ip.

**Reply by PH5HP** [reply]
Hi,
If I download myself I always see it in the logs. But last night I saw someone downloading something, but I saw nothing in the Logs section. On the other hand I see sometimes downloads in the logs. Also are the images of my layout still registered as downloads in the Logs section every time. Are they als counted as a download in the total download counter?

Kind regards,
Henk - https://fileserver.ph5hp.nl

**Comment by rejetto** [comment]
it's done and will be part of 0.55

**Comment by rejetto** [comment]
published :)

**Comment by PH5HP** [comment]
Hi,

Nice work in the monitor section. I would like that if something is downloaded that the row stays visible in the log and if another file is downloaded that the other row scrolls one down. Now I can also see all the traffic, but maybe only downloads and uploads in the left screen?

Now the row vanishes after the download has been completed.

Kind regards,
Henk - https://fileserver.ph5hp.nl

**Comment by rejetto** [comment]
when the download is finished in the left panel, you get a row in the right panel (live log).
when another download is finished, you get 2 rows in the live love, the first row doesn't vanish.
the live log is cleared when you change page.

**Reply by PH5HP** [reply]
Hi,

IYes, I saw that :-) ... I would like only to track downloads and uploads. I know they can be found in the the text-log-files, but it would be nice to have a list in the interface too.

Kind regards,
Henk

**Comment by rejetto** [comment]
at the moment the live log includes all that's included in the log, so it depends on your log options.
you may configure log options to only include download/uploads.
is that enough, or do you want to have only download/uploads in the live-log, and more things in the actual log?

i see in your screenshot that you enabled "log interface loading". It is disabled by default, as i think it's just noise for most people. Did you enable it purposely?

**Reply by PH5HP** [reply]
Hi,

I was thinking uploads and downloads somewhere in a history list. This way we can see what people mostly download.

No. I will turn 'log interface loading' off.

Kind regards,
Henk - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
> I was thinking uploads and downloads somewhere in a history list. This way we can see what people mostly download.

that doesn't seem to answer my question, but i'm not sure what you mean by "see what people mostly download". That sounds like something in the long term, more keen to what the "logs" page is. 
Monitoring is a page dedicated to what's going on in this very moment. if you are searching for statistics, we'll not have them here.

**Reply by PH5HP** [reply]
Hi,

Yes, that is what I am looking for... Download just scrolling down one row when they are completed if a new one is started... Some kind of history...

Kind regards,
Henk - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
but that's already happening, isn't it?
you are complaining that you see too much things, and not only downloads/uploads, right?
i guess it's best if it respects what you set here:
<img width="256" alt="image" src="https://github.com/user-attachments/assets/97a4f8da-e562-48cb-9bd8-a1057799b3aa">
